src/core/server/saved_objects/export/get_sorted_objects_for_export.test.ts

```diff
@@ -19,6 +19,12 @@
 
 import { getSortedObjectsForExport } from './get_sorted_objects_for_export';
 import { SavedObjectsClientMock } from '../service/saved_objects_client.mock';
+import { Readable } from 'stream';
+import { createPromiseFromStreams, createConcatStream } from '../../../../legacy/utils/streams';
```

Comment:
a bit concern with importing utils from legacy, but we haven't discuss this inside the team.

Reply:
I left these utilities in legacy when I moved Saved Objects to Core, so it's a todo for the Saved Objects migration. There are stable NPM packages for many of these so I'm hoping we can replace the utilities with dedicated npm dependencies.